contracts/Lens/JoeLensV2.sol

```diff
@@ -0,0 +1,364 @@
+            vars.underlyingTokenAllowance = account.balance;
+        } else {
+            JErc20 jErc20 = JErc20(address(jToken));
+            EIP20Interface underlying = EIP20Interface(jErc20.underlying());
+            vars.underlyingTokenBalance = underlying.balanceOf(account);
+            vars.underlyingTokenAllowance = underlying.allowance(account, address(jToken));
+        }
+
+        vars.jTokenBalance = jToken.balanceOf(account);
+        vars.borrowBalanceCurrent = jToken.borrowBalanceCurrent(account);
+
+        vars.balanceOfUnderlyingCurrent = jToken.balanceOfUnderlying(account);
+        PriceOracle priceOracle = joetroller.oracle();
+        uint256 underlyingPrice = priceOracle.getUnderlyingPrice(jToken);
+
+        (, uint256 collateralFactorMantissa, ) = joetroller.markets(address(jToken));
+
+        Exp memory supplyValueInUnderlying = Exp({mantissa: vars.balanceOfUnderlyingCurrent});
+        vars.supplyValueUSD = mul_ScalarTruncate(supplyValueInUnderlying, underlyingPrice);
+
+        Exp memory collateralFactor = Exp({mantissa: collateralFactorMantissa});
+        vars.collateralValueUSD = mul_ScalarTruncate(collateralFactor, vars.supplyValueUSD);
+
+        Exp memory borrowBalance = Exp({mantissa: vars.borrowBalanceCurrent});
+        vars.borrowValueUSD = mul_ScalarTruncate(borrowBalance, underlyingPrice);
+
+        return vars;
+    }
+
+    /**
+     * @notice Get an account's limits
+     * @param joetroller The joetroller address
+     * @param account The account who's limits are being retrieved
+     * @return An account's limits
+     */
+    function getAccountLimits(Joetroller joetroller, address account) public returns (AccountLimits memory) {
+        AccountLimits memory vars;
+        uint256 errorCode;
+
+        (errorCode, vars.liquidity, vars.shortfall) = joetroller.getAccountLiquidity(account);
+        require(errorCode == 0, "Can't get account liquidity");
+
+        vars.markets = joetroller.getAssetsIn(account);
+        JTokenBalances[] memory jTokenBalancesList = jTokenBalancesAll(vars.markets, account);
+        for (uint256 i = 0; i < jTokenBalancesList.length; i++) {
+            vars.totalCollateralValueUSD = add_(vars.totalCollateralValueUSD, jTokenBalancesList[i].collateralValueUSD);
+            vars.totalBorrowValueUSD = add_(vars.totalBorrowValueUSD, jTokenBalancesList[i].borrowValueUSD);
+        }
+
+        Exp memory totalBorrows = Exp({mantissa: vars.totalBorrowValueUSD});
+
+        vars.healthFactor = vars.totalCollateralValueUSD == 0 ? 0 : vars.totalBorrowValueUSD > 0
+            ? div_(vars.totalCollateralValueUSD, totalBorrows)
+            : 100;
+
+        return vars;
+    }
+
+    /**
+     * @notice Admin function to set new reward distributor address
+     * @param newRewardDistributor The address of the new reward distributor
+     */
+    function _setRewardDistributor(address payable newRewardDistributor) public {
```

Comment:
Prepend params with underscore

Reply:
Also, keep "_functionName" for private or internal functions only,
This one should be `setRewardDistributor`

Reply:
Gotcha, I'll also change our internal functions to use "_functionName" format